types/lib/model.d.ts

```diff
@@ -1961,7 +1969,7 @@ export abstract class Model<TModelAttributes extends {} = any, TCreationAttribut
   public static findAndCountAll<M extends Model>(
     this: ModelStatic<M>,
     options: SetRequired<FindAndCountOptions<M['_attributes']>, 'group'>
-  ): Promise<{ rows: M[]; count: number[] }>;
+  ): Promise<{ rows: M[]; count: Array<GroupedCountResultItem> }>;
```

Comment:
Tiny nit - I think just using `GroupedCountResultItem[]` is a bit more succinct, and is used by most of the rest of this file.